src/current/v23.1/vault-db-secrets-tutorial.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: Using HashiCorp Vault's Dynamic Secrets for Enhanced Database Credential Security in CockroachDB
+title: Use Vault to Securly Manage Database Credentials
```

Comment:
```suggestion
title: Use Vault to Securely Manage Database Credentials
```

Reply:
Part of me feels like "Securely" isn't even needed in this title, but I almost didn't write this comment — so I'm split on it.

Reply:
It's a fair point. I think we could go with `Use Hashicorp Vault to Manage Database Credentials` to also bring in Shannon's feedback about retaining "Hashicorp".

Reply:
Just being pedantic... I think it's "HashiCorp"